src/components/TopNav.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+
+function TopNav(props) {
+
+  function noHash(e){
+    e.preventDefault();
+    alert("hi");
+  }
+
+  return (
+
+    <div className="top-nav">
+      <ul>
+        <li><a href="#" onClick={//()=>{
+          //props.changeCategory("headlights");
+          noHash
+        }>Car Lights</a></li>
+
+        <li><a href="#" onClick={()=>{
+          props.changeCategory("tires");
+          noHash;
```

Comment:
lines 20 -21 are inside an arrow function where you need to type only valid code
typing the name of a function noHash doesn't accomplish anything
executing the function would accomplish something